### PR TITLE
Issue278 enh

### PR DESCRIPTION
**Pull request by JMox**
Issue #278

- Changed the "/help" slash command to "/cmdlist" or "/commandlist" 
- Added information to each slashcommand we currently have. 
- This is now displayed in it's own separate dialog box and the server console. 
- Added in a check for access_level=0 for @broxen

### CODE REVIEW

File: Projects/CoX/Servers/MapServer/SlashCommand.cpp [hunk 1]
```diff
@@ -513,19 +513,30 @@ void cmdHandler_SetU6(QString &cmd, Entity *e) {
...
         int alvl = getAccessLevel(*e);
         if(alvl >= sc.m_required_access_level)
-            msg += "\t" + sc.m_valid_prefixes.join(", ") + "\n";
+        {
+            // We don't want to show acess_level 0 (Under the Hood commands)
+            if(sc.m_required_access_level != 0 )
+            {
+                // Use msg for std out, msg_dlg for ingame dialog box
+                msg += "\t" + sc.m_valid_prefixes.join(", ") + "[" + QString::number(sc.m_required_access_level) + "]:\t" + sc.m_help_text + "\n";
+                msg_dlg += "<color #ffCC99><i>" + sc.m_valid_prefixes.join(", ") + "</i></color>[<color #66ffff>" + QString::number(sc.m_required_access_level) + "</color>]: " + sc.m_help_text + "<br>";
```
Comment (nemerle) [inline Projects/CoX/Servers/MapServer/SlashCommand.cpp]:
```cpp
QString("<color #ffCC99><i>%1</i></color>[<color #66ffff>%2</color>]: %3<br>").arg(sc.m_valid_prefixes.join(", ")).arg(sc.m_required_access_level).arg(sc.m_help_text)
```
Might be more readable ?

Comment (JMox) [reply Projects/CoX/Servers/MapServer/SlashCommand.cpp]: Can do, not sure if the formatting will change much though?

Comment (nemerle) [reply Projects/CoX/Servers/MapServer/SlashCommand.cpp]: It's mostly for code compactness :P
Also, the smlx/html-ish code is easier to check visually if it's wrapped in 1 string :)

Comment (JMox) [reply Projects/CoX/Servers/MapServer/SlashCommand.cpp]: Ah kk! 
I'll go ahead and change this as well as update the motd script.

**Pull request merged**